test/test_ops.py

```diff
@@ -1,15 +1,18 @@
 from __future__ import division
+import math
+from typing import Tuple
```

Comment:
Can you import from `torch.jit.annotations`, instead of `typing`? Same reason as below